src/content/docs/en/install/auto.mdx

```diff
@@ -1,154 +1,424 @@
 ---
-title: Install Astro with the Automatic CLI
-description: 'How to install Astro with NPM, PNPM, or Yarn via the create-astro CLI tool.'
-i18nReady: true
+title: Install Astro
+description: A reference page in my new Starlight docs site.
 ---
-import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+import { Tabs, TabItem, FileTree, CardGrid, LinkCard, Steps } from '@astrojs/starlight/components';
+import ReadMore from '~/components/ReadMore.astro';
 
+The [`create astro` CLI command](#install-from-the-cli-wizard) is the fastest way to start a new Astro project from scratch. It will walk you through every step of setting up your new Astro project and allows you to choose from a few different official starter templates. 
```

Comment:
Not 100% sure, but I suppose here would be `allow` instead?
```suggestion
The [`create astro` CLI command](#install-from-the-cli-wizard) is the fastest way to start a new Astro project from scratch. It will walk you through every step of setting up your new Astro project and allow you to choose from a few different official starter templates. 
```

Reply:
Oooh, interesting! I guess if it stems from "It will..." then it's "allow". If it's "It ... (will walk you through)" then it's "allows". 

Quick gut check says "allow" reads better, so let's go with that!